engine-jdk8-tests/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/JavaFXClassLoadingTest.java

```diff
@@ -20,6 +19,8 @@
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.junit.Test;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
```

Comment:
We try to put the import static at the top (it's not yet consistent in the code but we fix it when we update a file so we might as well not introduce any more inconsistent code). Could you configure your IDE to do so and fix the updated file?